game-sales-tracker/src/app/shared/validators/password.validator.ts

```diff
@@ -0,0 +1,26 @@
+import { AbstractControl } from "@angular/forms";
+
+export function PasswordValidate(control: AbstractControl): { [key: string]: boolean } {
+    const password: string = control.value;
+    if (password === null || password === "") {
+        return { required: true };
+    }
+    if (password.length < 8) {
+        return { minLength: true };
+    }
+    if (password.length > 15) {
+        return { maxlength: true };
+    }
+    if (!(new RegExp(/[A-Z]/).test(password))) {
+        return { capitalCase: true };
+    }
+    if (!(new RegExp(/[a-z]/).test(password))) {
+        return { lowerCase: true };
+    }
+    if (!(new RegExp(/[0-9]/).test(password))) {
+        return { numbers: true };
+    }
+    if (!(new RegExp(/[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/).test(password))) {
+        return { specialCharacters: true };
+    }
```

Comment:
We don't need so difficult password, only letters and let it be one number, special symbols only if person want